recipes/jupyterlab-js-logs/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyterlab-js-logs" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 33fde10068487c7ee15ad114447250aff02f87c0130091f233a2b59d1fc47091
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - jupyter-packaging >=0.10,<2
+  run:
+    - jupyterlab >=3.0,<4
+    - python >=3.6
+
+test:
+  commands:
+    - pip check
+    - test -f "${PREFIX}/share/jupyter/labextensions/jupyterlab-js-logs/package.json"                                    # [unix]
+    - if exist %PREFIX%\\share\\jupyter\\labextensions\\jupyterlab-js-logs\\package.json (exit 0) else (exit 1)          # [win]
+  requires:
+    - pip
+
+about:
+  home: https://github.com/QuantStack/jupyterlab-js-logs
+  summary: JupyterLab extension to show the logs from the browser developer tools console
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jtpio
+    - hbcarlos
```

Comment:
cc @hbcarlos if you are fine to be listed here (since you are on the list of PyPI maintainers for https://pypi.org/project/jupyterlab-js-logs/)

Reply:
Sure. Go ahead.